src/Symfony/Component/AssetMapper/Tests/Factory/CachedMappedAssetFactoryTest.php

```diff
@@ -103,6 +104,7 @@ public function testAssetConfigCacheResourceContainsDependencies()
 
         // just adding any file as an example
         $mappedAsset->addFileDependency(__DIR__.'/../fixtures/importmap.php');
+        $mappedAsset->addFileDependency(__DIR__.'/../fixtures/dir3');
```

Comment:
Should we have a `addDirectoryDependency` instead to make the intent more clear ?

Reply:
Perhaps - though I'm also trying to keep the patch on the smaller side. 

We could merge this as-is, then think about a more generic method name like `addPathDependency()` for 6.4.